### ISSUE TITLE
A2C and ACKTR do not output their scores

### ISSUE THREAD

**Issue by MagiKoco** [openai/baselines#469]
A2C and ACKTR only print the information about loss function  (as follows) instead of the scores in a game. As a result, I wonder if it can work for Atari Games. I would like to know how to output their performance on a game. Thank you.    

            ev = explained_variance(values, rewards)
            logger.record_tabular("nupdates", update)
            logger.record_tabular("total_timesteps", update*nbatch)
            logger.record_tabular("fps", fps)
            logger.record_tabular("policy_entropy", float(policy_entropy))
            logger.record_tabular("value_loss", float(value_loss))
            logger.record_tabular("explained_variance", float(ev))
            logger.dump_tabular()

**Comment by DanielTakeshi** [comment]
Did you check out the pull requests?

https://github.com/openai/baselines/pull/322

**Comment by MagiKoco** [comment]
Thank you very much for your promptness! I'll try it.

**Comment by DanielTakeshi** [comment]
@MagiKoco Sorry for the long delay. are you still running into issues? I am getting 20+ scores for Pong and 400+ for Breakout using the refactored A2C code. Can provide details on hyperparameters if needed
